editor/utils/tracking.js

```diff
@@ -0,0 +1,61 @@
+/* eslint-disable no-console */
```

Comment:
We should avoid adding a lot of small packages in this repo.  This is what `WordPress/packages` is designed for; I'm fine with moving stats tracking over there once we get the build system working well.  We will probably want to make it a bit more generic (remove the `gutenberg_` prefix and modify the opt-in logic).